src/SFA.DAS.Commitments.Support.SubSite/Mappers/ApprenticeshipMapper.cs

```diff
@@ -0,0 +1,127 @@
+﻿using SFA.DAS.Commitments.Application.Queries.GetApprenticeshipsByUln;
+using SFA.DAS.Commitments.Domain.Entities;
+using SFA.DAS.Commitments.Support.SubSite.Extensions;
+using SFA.DAS.Commitments.Support.SubSite.Extentions;
+using SFA.DAS.Commitments.Support.SubSite.Models;
+using SFA.DAS.HashingService;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace SFA.DAS.Commitments.Support.SubSite.Mappers
+{
+    public class ApprenticeshipMapper : IApprenticeshipMapper
```

Comment:
Doing too much.  Not well tested.  Generally a bit awful.  I don't think a mapper should take in any dependencies.